Wiki/Modules/FvwmBacker/index.md

```diff
@@ -3,14 +3,94 @@ title: FvwmBacker
 type: module
 weight: 430
 description: |
-  This module will change the backgound/wallpaper when changing desktops.
+  This module will change the background/wallpaper when changing desktops.
   Any command can be executed when changing desktops so this module can
   be used for more than just switching the background.
 ---
 # FvwmBacker
 
-The FvwmBacker module provides functionality to change the  background when
-changing  desktops.  Any command can be executed to change the backgrounds.
-Actually, any arbitrary command can be sent to fvwm to execute, so  you
+* TOC
+{:toc}
+
+The FvwmBacker module provides functionality to change the background when
+changing  desktops. Any command can be executed to change the backgrounds.
+Actually, any arbitrary command can be sent to fvwm to execute, so you
 could also do things such as changing window border colors, etc.
 
+You can use the FvwmPager to move from screen to screen through each virtual
+desktop set. Now that you can move easily between the desktops you can further
+personalize them by giving them individual background colors.
+
+{% fvwm2rc %}
+*FvwmBacker: Command (Desk 2) -solid white
+{% endfvwm2rc %}
+
+The changes will not be implemented until you instruct Fvwm to update the screens
+during the initialization:
+
+{% fvwm2rc %}
+AddToFunc InitFunction I Module FvwmBacker
+{% endfvwm2rc %}
+
+Instead of colors, you can define background images:
+
+{% fvwm2rc %}
+*FvwmBacker: Command (Desk 0) Exec exec fvwm-root $HOME/background-mono.png
+*FvwmBacker: Command (Desk 1) Exec exec fvwm-root $HOME/wallpaper-nature.jpg
+{% endfvwm2rc %}
+
+## Example
+
+You can set each desktop screen (or page) with a different background. This
+advanced example changes your Desktop with 4 desks and FvwmPager miniature
+views with different images.
+
+![image](multidesk-fvwmbacker.png){:.d-block .mx-auto .img-fluid}
+
+For FvwmPager configuration, refer to the
+[FvwmPager]({{ "/Modules/FvwmPager" | prepend: site.wikiurl }}) page. Images
+for the miniature views you configured with Colorsets using Pixmap. For colors,
+you can also use RootTransparent.
+
+{% fvwm2rc %}
+## Transparent tint color
+Colorset 22 RootTransparent Buffer, fg black, Tint #8bade7 60
+## Thumbnail
+Colorset 23 Pixmap $HOME/.fvwm/images/email-thumb.png
+Colorset 28 Pixmap $HOME/.fvwm/images/work-thumb.png
+Colorset 25 Pixmap $HOME/.fvwm/images/misc-thumb.png
+{% endfvwm2rc %}
+
+**FvwmPager**
+{% fvwm2rc %}
+DesktopName 0 Web
+DesktopName 1 Email
+DesktopName 2 Work
+DesktopName 3 Misc
+
+DestroyModuleConfig FvwmPager: *
+*FvwmPager: Colorset 0 22
+*FvwmPager: Colorset 1 23
+*FvwmPager: Colorset 2 28
+*FvwmPager: Colorset 3 25
+{% endfvwm2rc %}
+
+**FvwmBacker**
+
+{% fvwm2rc %}
+DestroyModuleConfig FvwmBacker: *
+*FvwmBacker: Command (Desk 0, Page 0 0) Exec exec fvwm-root $HOME/.fvwm/images/web-tiled.png
+*FvwmBacker: Command (Desk 0, Page 1 0) Exec exec fvwm-root $HOME/.fvwm/images/web-tiled.png
```

Comment:
This is overkill, you are setting the exact same image for all pages, so just remove the `, Page X Y` bit. See above comment, just add a small comment stating that it is possible to control each pages, but this example doesn't need that.